linux_os/guide/system/logging/journald/service_systemd-journal-upload_enabled/tests/service_disabled.fail.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# packages = systemd
+
+{{{ bash_package_install("systemd-journal-upload") }}}
```

Comment:
The package name should be `systemd-journal-remote`

Reply:
File removed